lockup/src/lib.rs

```diff
@@ -295,6 +295,19 @@ mod tests {
         contract.add_staking_access_key(public_key(4));
     }
 
+    #[test]
+    #[should_panic(expected = "Can not remove signer public key")]
```

Comment:
```suggestion
    #[should_panic(expected = "Can not remove owner public key")]
```

Reply:
It still need to emphasize why you can't remove owner's public key. Basically, saying the key used to sign the transaction.

Reply:
Updated to `Can not remove owners public key that is used to sign this transaction`